### PR TITLE
Create troubleshoot-pi-adapter-for-opc-ua.md

### PR DESCRIPTION
**Pull request by TinaUll**
Please review for accuracy. Thank you!

### CODE REVIEW

File: V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+uid: TroubleshootPIAdapterForOPCUA
+---
+
+# Troubleshoot PI Adapter for OPC UA
+
+If you encounter issues with PI Adapter for OPC UA, you can attempt to troubleshoot on your own before contacting OSIsoft Technical Support. Start by checking the adapter's configuration, connectivity, and logs. The following sections provide more details.
+
+## Check configurations
+
+1. In the [data source configuration](xref:PIAdapterForOPCUADataSourceConfiguration), verify the accuracy of the configured **EndpointURL** and **RootNodeIds**.
```
Comment (zryska) [inline V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md:11]: would correctness be better word than accuracy here?

Comment (zryska) [inline V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md:11]: We might also add that users should check  **userName** and **password** when specified.

**Review by zryska — APPROVED** [review]

File: V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md (hunk 1 above)
Comment (TinaUll) [reply V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md]: Rephrased the whole sentence.

Comment (TinaUll) [reply V1.2/Troubleshooting/troubleshoot-pi-adapter-for-opc-ua.md]: Done!

**Pull request merged**